tests/testthat/test-getBioEntityInfoFromIndra.R

```diff
@@ -0,0 +1,59 @@
+# Test .callGetUniprotIdsFromUniprotMnemonicIdsApi
```

Comment:
Can we rename this file to test_utils_annotateProteinInfoFromIndra?

Reply:
done